### ISSUE TITLE
Ordering of Functions/Classes

### ISSUE THREAD

**Issue by miraydamartinez** [emad0525/pwp-capstones#2]
Typically we define classes at the start of our code because they provide the foundation of the program that you use throughout. Your **class TextSample:** is all the way at the bottom when it should appear in the **Creating the Definition for our Model** section.

**Comment by emad0525** [comment]
Makes sense. The only reason I moved to the bottom is because of the instructions provided. 

"If you need to restart your Jupyter Notebook either run the cells below first or move the TextSample class definition & instantiation cells to the bottom."

Perhaps, I misunderstood the instructions. Thanks for the clarification.
